devices.js

```diff
@@ -6692,6 +6692,19 @@ const devices = [
         description: 'Zigbee LED controller dimmer',
         extend: preset.gledopto.light_onoff_brightness,
     },
+    {
+        zigbeeModel: ['GL-C-009S'],
+        fingerprint: [
```

Comment:
I don't expect this `fingerprint` to be needed (`zigbeeModel`) is enough, can you try removing it